src/neuroconv/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, file_path: FilePathType, verbose: bool = True):
 
         hdf5storage = get_package(package_name="hdf5storage")
 
-        super().__init__(spikes_matfile_path=file_path, verbose=verbose)
+        super().__init__(file_path=file_path, verbose=verbose)
```

Comment:
Oh, also, issue a deprecation warning on NeuroConv side

Reply:
Good idea. 

Here is the file:
https://www.dropbox.com/s/q4i4dsnk7rqh09h/Peter_MS22_180629_110319_concat.spikes.cellinfo.mat?dl=0

Maybe you can drop the other columns to make it smaller so we can add it to the gin tests?

Reply:
Wait, I did something possibly dumb here. Can you check out the new code and see if this is what you intended? You already had the right input `file_path` and that is only handled internally.

Reply:
Sorry, I mean the converted NWB file from using this interface (possibly on that file, but also maybe other GIN files too)

Reply:
What did you mean about the deprecation though? What should be deprecated?